php/class-photonfill.php

```diff
@@ -378,6 +382,26 @@ public function image_size_names_choose( $data ) {
 		}
 
 		/**
+		 * Add image caption requires element to have a width to display shortcode correctly
+		 */
+		public function add_width_for_captions( $html, $id, $caption, $title, $align, $url, $size, $alt = '' ) {
+			$caption = apply_filters( 'image_add_caption_text', $caption, $id );
+			if ( ! empty( $caption ) ) {
+				if ( is_numeric( $size ) ) {
+					$size_px = $size;
+				} elseif ( is_array( $size ) ) {
+					$size_px = $size[0];
+				} else {
```

Comment:
owen and I discussed, but there are many break points for a size so what width is selected is irrelevant as the polyfill will pick whatever works best for the window. This is really just a workaround for the fact that images require a width or else the caption is not set.